README.md

```diff
@@ -88,7 +88,7 @@ var config = KafkaConsumerConfiguration(
 )
 config.bootstrapServers = [broker]
 
-let consumer = try KafkaConsumer(
+let consumer = try KafkaConsumer.makeConsumer(
```

Comment:
Overall, I still think we should make the `init()` public so that in the case where users are not interested in `event` they can just call `KafkaProducer()` and `KafkaConsumer()`.

Reply:
Right, also applied this to `KafkaProducer.makeProducer`